config/wado.js

```diff
@@ -0,0 +1,122 @@
+module.exports = function (doc) {
```

Comment:
Maybe add a comment here about the language restrictions and any other requirements?